itests/fevm_test.go

```diff
@@ -139,3 +141,78 @@ func TestEVMRpcDisable(t *testing.T) {
 	_, err := client.EthBlockNumber(context.Background())
 	require.ErrorContains(t, err, "module disabled, enable with Fevm.EnableEthRPC")
 }
+
+// TestFEVMRecursiveFuncCall deploys a contract and makes a recursive function calls
+func TestFEVMRecursiveFuncCall(t *testing.T) {
+	ctx, cancel, client := setupFEVMTest(t)
+	defer cancel()
+
+	//install contract Actor
+	filenameActor := "contracts/StackFunc.hex"
+	fromAddr, actorAddr := client.EVM().DeployContractFromFilename(ctx, filenameActor)
+
+	testN := func(n int) func(t *testing.T) {
+		return func(t *testing.T) {
+			inputData := make([]byte, 32)
+			binary.BigEndian.PutUint64(inputData[24:], uint64(n))
+
+			result := client.EVM().InvokeContractByFuncName(ctx, fromAddr, actorAddr, "exec1(uint256)", inputData)
+			require.Equal(t, result, []byte{})
+		}
+	}
+
+	t.Run("n=0", testN(0))
+	t.Run("n=1", testN(1))
+	t.Run("n=20", testN(20))
+	t.Run("n=200", testN(200))
+	t.Run("n=293", testN(293))
```

Comment:
Is anything special about 293? Would there be any benefit to also testing to exit code 23 in this test?